pkg/apis/rollouts/v1alpha1/analysis_types.go

```diff
@@ -225,3 +226,38 @@ type Measurement struct {
 	// ResumeAt is the  timestamp when the analysisRun should try to resume the measurement
 	ResumeAt *metav1.Time `json:"resumeAt,omitempty"`
 }
+
+type KayentaMetric struct {
+	Address string `json:"address"`
+
+	Application string `json:"application"`
+
+	CanaryConfigName string `json:"canaryConfigName"`
+
+	MetricsAccountName       string `json:"metricsAccountName"`
+	ConfigurationAccountName string `json:"configurationAccountName"`
+	StorageAccountName       string `json:"storageAccountName"`
+
+	Threshold KayentaThreshold `json:"threshold"`
+
+	Scopes []KayentaScope `json:"scopes"`
+}
+
+type KayentaThreshold struct {
+	Pass     int `json:"pass"`
+	Marginal int `json:"marginal"`
+}
+
+type KayentaScope struct {
+	Name            string      `json:"name"`
+	ControlScope    ScopeDetail `json:"controlScope"`
+	ExperimentScope ScopeDetail `json:"experimentScope"`
+}
+
+type ScopeDetail struct {
+	Scope     string `json:"scope"`
+	Region    string `json:"region"`
+	Step      int    `json:"step"`
+	StartTime string `json:"start"`
+	EndTime   string `json:"end"`
```

Comment:
Code generation is reporting two API violations for StartTime and EndTime because the json struct tags do not match the field names. They need to match.